definitions/output/reports/dynamic_publisher.js

```diff
@@ -0,0 +1,16 @@
+const configs = new reports.HTTPArchiveReports()
+const params = {
+  date: constants.currentMonth,
+  rankFilter: constants.devRankFilter
+}
+
+const metrics = configs.listMetrics()
+metrics.forEach(metric => {
+  metric.SQL.forEach(sql => {
+    publish(sql.type, {
+      type: 'table',
+      schema: 'reports',
+      tags: ['crawl_reports']
+    }).query(ctx => constants.fillTemplate(sql.query, params))
```

Comment:
In `reports` dataset we could store intermediate aggregated data - it's easier to check for data issues in BQ than in GCS.
Cloud Function then will pick fresh rows and save them to GCS.

I think we could be fine with a table per chart type, e.g `httparchive.reports.timeseries`:
- date (partition)
- metric (cluster)
- timestamp
- client (cluster)
- p10
- p25
- p50
- p75
- p90